proto/arkeo/claim/query.proto

```diff
@@ -0,0 +1,44 @@
+syntax = "proto3";
+
+package arkeonetwork.arkeo.claim;
+
+import "gogoproto/gogo.proto";
+import "google/api/annotations.proto";
+import "cosmos/base/query/v1beta1/pagination.proto";
+import "arkeo/claim/params.proto";
+
+option go_package = "github.com/arkeonetwork/arkeo/x/claim/types";
+
+// Query defines the gRPC querier service.
+service Query {
+  
+  // Parameters queries the parameters of the module.
+  rpc Params (QueryParamsRequest) returns (QueryParamsResponse) {
+    option (google.api.http).get = "/arkeonetwork/arkeo/claim/params";
+  
+  }
+  
+  // Queries a list of ClaimRecord items.
+  rpc ClaimRecord (QueryClaimRecordRequest) returns (QueryClaimRecordResponse) {
+    option (google.api.http).get = "/arkeonetwork/arkeo/claim/claim_record/{address}";
```

Comment:
[optional]it is not recommended to use `_` in url , usually that's for SEO reasons , because google doesn't like it.

Maybe we follow it as a good practice as well?